File: app/src/main/java/br/vino/transmobisp/model/Stop.kt
package br.vino.transmobisp.model

import java.io.Serializable

data class Stop(
    val cp: Int, // Stop code
    val np: String, // Stop name
    val ed: String, // Address
    val py: Double, // Latitude
    val px: Double // Longitude
) : Serializable

File: sp_movement/lib/app/modules/estimated_arrival/models/domain/stop.dart
import 'localized_lines.dart';

class Stop {
  int code;
  String name;
  double latitude;
  double longitude;
  List<LocalizedLines> localizedLines;

  Stop({
    required this.code,
    required this.name,
    required this.latitude,
    required this.longitude,
    required this.localizedLines,
  });
}
